test/ulid.spec.js

```diff
@@ -144,19 +144,19 @@ describe("ulid", function() {
                 });
 
                 it("first call", function() {
-                    expect(this.ulid()).to.equal("01ARYZ6S41YYYYYYYYYYYYYYYY");
+                    expect(this.ulid()).to.equal("01ARYZ6S41NNNNNNNNNNNNNNNN");
```

Comment:
Any idea why these changed? I don't see it from glancing at the changes you made.. But I suspect it's [this line](https://github.com/perry-mitchell/ulidx/pull/15/files#diff-1fabd60c86d54858595dc28a7a6495c94000b3fb49b77623f14f619664d03f59R107) doing it.. Thoughts?

I'd ideally like to leave these unchanged as a sign of stability.

Reply:
Brought back the previous implementation, no performance improvements but kept stability as wanted :)